inventory

```diff
@@ -1,10 +1,11 @@
 [masters]
-#host1 ansible_user=giorgia ansible_ssh_user=giorgia ansible_ssh_host=192.168.1.23 ansible_ssh_private_key_file=~/.ssh/id_rsa
-host1 ansible_user=giorgia ansible_host=192.168.1.23 
+127.0.0.1 ansible_connection=local
```

Comment:
@giovanni I would suggest not to touch anymore the "inventory" file. Mayeb, you can create something like "inventory.giovanni", and rename it as "inventory" when you have to use it locally. Of course, the same will apply also for Giorgia and Sara.

Reply:
Understood, I'll revert this change.
I would just to point out that this inventory file configuration is required by the script to work.
The script clones the git repository locally, and then, executes the ansible-playbook with this inventory file that has as the target machine the local one.